### PR TITLE
Azure Deployment

### PR DESCRIPTION
**Pull request by priyankarki**
Readme describing Azure deployment steps, as well as the build .yml file to run within Github Actions

### CODE REVIEW

**Comment by princiya** [comment]
thanks for the PR @priyankarki .
I will let @sunithapatel review this :)

File: deployments/azure/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+This is a GitHub Action yml file that will deploy the current React web application to [Azure Static Web Apps](http://<link to doc>). It is placed here as there is no Azure account to tie it to just yet. 
+
+### Setup
+To set up an Azure Static Web App using GitHub, follow the directions highlighted in [this](https://docs.microsoft.com/en-us/azure/static-web-apps/get-started-portal?tabs=vanilla-javascript)
+documentation article.
+
+```
+1. Set up your static web app repository on Github
+2. Create an account on the Azure portal
+3. Create a new project using the Azure portal.
```
Comment (sunithapatel) [inline deployments/azure/readme.md:11]: When you say “create new project” is that for Static Web App resource in Azure portal? If so, can you mention that here?

File: deployments/azure/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+4. Fill in the details of your project, and link your Github account to Azure.
+5. Select "create" after which the build .yml files will be created in your 
+Github repository and begin deplyment using Github Actions.
```
Comment (sunithapatel) [inline deployments/azure/readme.md:14]: Spelling of `deployment` 🙂

Comment (sunithapatel) [inline deployments/azure/readme.md:14]: Also after this last statement where you say build.yml will be created, can you say something like `The yml file in this folder is an example of the file that will get generated.`

**Comment by sunithapatel** [comment]
@priyankarki Sorry for my late reply on this! Thank you so much for doing this! Hope you could learn from it too. 🙂
I put a few small things to address, but overall great work! 👍🏼

File: deployments/azure/readme.md (hunk 2 above)
Comment (priyankarki) [inline deployments/azure/readme.md:14]: deployment typo

**Comment by priyankarki** [comment]
@sunithapatel I made the edits you suggested, let me know if there's anything else!

**Review by sunithapatel — APPROVED** [review]

**Comment by sunithapatel** [comment]
Great work @priyankarki! Thank you for doing this!

**Pull request merged**